src/util.cc

```diff
@@ -449,6 +449,69 @@ void Win32Fatal(const char* function, const char* hint) {
     Fatal("%s: %s", function, GetLastErrorString().c_str());
   }
 }
+
+bool TryConvertUtf8ToAnsi(const char* utf8, string& ansi,
+                          unsigned int codepage) {
```

Comment:
Why have a codepage parameter? ansi is always going to be the result of `GetACP`. If any other value is passed as an argument, the method becomes a misnomer.

Reply:
The codepage is there for testing, so that we can force non-active code pages to induce conversion failures. It's commented in the header, but I'll add a comment here too for completeness.